test_infra/src/lib.rs

```diff
@@ -1298,7 +1298,10 @@ impl<'a> GuestCommand<'a> {
 
 pub fn clh_command(cmd: &str) -> String {
     env::var("BUILD_TARGET").map_or(
-        format!("target/x86_64-unknown-linux-gnu/release/{}", cmd),
+        #[cfg(target_arch = "aarch64")]
+        format!("target/aarch64-unknown-linux-gnu/release/{}", cmd),
+        #[cfg(target_arch = "x86_64")]
+        format!("target/aarch64-unknown-linux-gnu/release/{}", cmd),
```

Comment:
This should be x86_64 toolchain

Reply:
FWIW: Changes should not be needed as the env variable `BUILD_TARGET` was set correctly from the `run_metrics.sh` script. This is also why the integration test works